{{ cookiecutter.repo_name }}/.cookiecutter/config

```diff
@@ -3,3 +3,33 @@ export PROJECT_NAME={{ cookiecutter.project_name }}
 export REPO_NAME={{ cookiecutter.repo_name }}
 export DESCRIPTION={{ cookiecutter.description }}
 export PROJECT_OPENNESS={{ cookiecutter.openness }}
+export ENVIRONMENT_TYPE={{ cookiecutter.venv_type }}
+
+{% if cookiecutter.venv_type == 'conda' %}
+# Conda configuration
+ENVIRONMENT_NAME=${REPO_NAME}
```

Comment:
Oh, this is a really nice approach. Maybe since `ENVIRONMENT_NAME` isn't actually called later on in the install (and isn't really saving keystrokes or improving readability), just use `${REPO_NAME}` directly below? Same with `VENV_PATH`. That way all the defined actions are only those called later in the install.